test/check-openshift

```diff
@@ -34,6 +34,7 @@ class TestOpenshift(MachineCase):
 
         tmpfile = os.path.join(self.tmpdir, "config")
         self.openshift.download("/root/.kube/config", tmpfile)
+        self.openshift.upload(["files/mock-app-openshift.json"], "/tmp")
```

Comment:
It seems that the upload should happen to the test machine, rather than the openshift machine, no?